test/e2e/iam/projects_test.go

```diff
@@ -90,6 +93,33 @@ func TestProjects(t *testing.T) {
 		require.NoError(t, err, string(resp))
 	})
 
+	t.Run("Tags", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			projectsEntity,
+			"describe",
+			projectID,
+			"-o=json")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+		require.NoError(t, err, string(resp))
+		var project atlasv2.Group
+		if err := json.Unmarshal(resp, &project); err != nil {
+			t.Fatalf("unexpected error: %v", err)
+		}
+
+		require.Len(t, project.GetTags(), 2)
+
+		expectedTags := map[string]string{"env": "e2e", "prod": "false"}
+		for _, tag := range project.GetTags() {
+			expectedValue, ok := expectedTags[tag.GetKey()]
+			if !ok {
+				t.Errorf("expected %v, to have key %s\n", project.Tags, tag.GetKey())
+			}
+
+			require.Equal(t, expectedValue, tag.GetValue())
+		}
```

Comment:
what about: 
```
		expectedTags := []atlasv2.ResourceTag{
			{Key: pointer.Get("env"), Value: pointer.Get("e2e")},
			{Key: pointer.Get("prod"), Value: pointer.Get("false")},
		}
		require.Equal(t, expectedTags, project.GetTags())
```

Reply:
There's no guarantee that the server will return the tags in the same order, so I don't think we can use this approach.